dashboard/test/controllers/api/v1/projects/public_gallery_controller_test.rb

```diff
@@ -13,51 +13,25 @@ class Api::V1::Projects::PublicGalleryControllerTest < ActionController::TestCas
       studentAgeRange: "18+",
       isFeatured: true
     }
-    all_featured_projects = {applab: [published_applab_project], gamelab: [], spritelab: [], playlab: [], artist: [], minecraft: [], events: [], k1: [], dance: [], poetry: [], library: []}
-    applab_featured_projects = {applab: [published_applab_project]}
-    ProjectsList.stubs(:fetch_featured_published_projects).returns(all_featured_projects)
-    ProjectsList.stubs(:fetch_featured_projects_by_type).returns(applab_featured_projects)
+    all_featured_projects = {applab: [published_applab_project], gamelab: [], spritelab: [], playlab: [], artist: [], minecraft: [], events: [], k1: [], dance: [], poetry: [], library: [], music: []}
+    applab_featured_projects = [published_applab_project]
+    ProjectsList.stubs(:fetch_active_published_featured_projects).returns(all_featured_projects)
   end
 
-  test_user_gets_response_for(
```

Comment:
Can you elaborate on these deletions?

Reply:
Since `fetch_active_published_featured_projects` is being stubbed to return `all_featured_projects`, and `index` is returning
```
`render json: ProjectsList.fetch_active_published_featured_projects(params[:project_type])
```
these unit tests were returning the same response despite different `params` so not testing what they were intended to test. Therefore, I decided to remove.
Thanks for asking about this!